.hemtt/project.toml

```diff
@@ -34,5 +34,17 @@ preset = "Hemtt"
 
 [hemtt.launch]
 workshop = [
-    "450814997", # CBA_A3's Workshop ID
+    "450814997", # CBA_A3
+]
+
+[hemtt.launch.spe]
+workshop = [
+    "450814997", # CBA_A3
+    "2369477168", # ADT
```

Comment:
forgot to remove, but honestly I think this should stay, and go in the others as well. If the person doesn't have the mod, it just won't load, no problems

Reply:
Actually maybe I should add `workshop_optional` or something, and warn people when mods don't exists...